.ci/test_r_package_windows.ps1

```diff
@@ -48,11 +48,15 @@ Write-Output "Installing Rtools"
 Start-Process -FilePath Rtools.exe -NoNewWindow -Wait -ArgumentList "/VERYSILENT /DIR=$env:R_LIB_PATH/Rtools" ; Check-Output $?
 Write-Output "Done installing Rtools"
 
+Write-Output "Installing dependencies"
+$packages = "c('data.table', 'httr', 'jsonlite', 'Matrix', 'processx', 'R6', 'testthat'), dependencies = c('Imports', 'Depends', 'LinkingTo')"
```

Comment:
We don't need `httr` package for non-MinGW builds. It just increases installation time and the number of installed packages that potentially can conflict with each other. Also, it is technically not a `dependency` of LightGBM (refer to the log message above).

Reply:
oh true! I'm going to move an `install.packages("httr")` into the script downloading MiKTeX. Something like

```r
if (!require("httr")){
    install.packages("httr")
}
```

That way, we won't have to introduce another `if ($env:COMPILER -ne "MINGW")` stuff in `test_r_package_windows.ps1`.

Reply:
I like your idea!